content/documentation/pages/5-feature-guides/2-batch/6-composed-task.md

```diff
@@ -0,0 +1,643 @@
+1. `transition-sample.exitMessage` - the property itself.
+
+More can be read about the sections of a property and the different property types [here](#passing-properties)
+
+#### Configuring your split
+
+In the example above we configured the behavior of our split in the composed task by using the `spring.cloud.task.closecontextEnabled` and `split-thread-core-pool-size` properties. Let's look at all the properties you can take advantage of when using splits.
+
+- spring.cloud.task.closecontextEnabled - When using splits, this property is required to be set to `true` because the context will not close, because threads were allocated to support the split.
+- split-thread-core-pool-size - Establishes the initial number of threads required for the splits in the composed task. Each child task contained in a split requires a thread in order to execute. (Defaults to 1)
+- split-thread-max-pool-size - The maximum number threads to be allocated.
+- split-thread-queue-capacity - The number of tasks that should be enqueued if all threads are in use before a new thread is allocated.
+
+##### Basic Split Sizing
+
+For a basic setup using just the `split-thread-core-pool-size` property. You want to look at your graph and count the split that has the largest number of child tasks,
+this will be the number of threads you will need to utilize. To set the thread count use the split-thread-core-pool-size property (defaults to 1).
+So for example a definition like: <AAA || BBB || CCC> && <DDD || EEE> would require a split-thread-core-pool-size of 3.
+This is because the largest split contains 3 child tasks. A count of 2 would mean that AAA and BBB would run in parallel but CCC would wait until either AAA or BBB to finish in order to run. Then DDD and EEE would run in parallel.
+
+##### Sizing for large splits
+
+In cases where there are a large number of tasks in a split or you have tight restrictions on threads you can dynamically adjust the thread allocations.
+Let' say we have 2 splits in our composed task:
+
+1. split-a has 5 tasks
+1. split-b has 50 tasks
+
+So lets look at an example on how we could set the pool sizing:
+
+1. Set the `split-thread-core-pool-size` We could set this size to 5 threads to support split-a.
+1. Set the `split-thread-max-pool-size` Let's say we want to limit the number of executing tasks to 10. We could set this value to 10. This means that for split-b no more than 10 tasks would be executed.
+1. Set the `split-thread-queue-capacity` Allows you to configure the number of tasks should be enqueued in a split prior to a new thread to be allocated. Since this is defaulted to `MAX_VALUE`, no new threads would be allocated. We could set this value to 5.
+
+This will require testing to find the right sizing. One sizing strategy does not fit all.
+
+## Restarting Composed Task Runner when child task fails
+
+Composed tasks in Spring Cloud Data Flow allow users to re-launch the failed composed task in cases where a child task fails.
```

Comment:
keep noticing some use of 'child task'.... the meaning isn't so clear really.  if everything in the DAG is a 'child' then we should remove the used of the word child...